README.md

```diff
@@ -94,6 +94,11 @@ Note, you will also need to run an upstream tool (eg, https://github.com/ONSDigi
 docker run -it -p 8000:8000 onsdigital/go-launch-a-survey:latest
 ```
 
+You will also need to run a Survey Register for Survey Runner to load Schemas from 
+```
+docker run -it -p 8080:8080 onsdigital/eq-survey-register:simple-rest-api
+```
```

Comment:
The survey register isn't required yet, as the existing production form types will still be in survey runner.